tempesta_fw/http.h

```diff
@@ -232,6 +232,7 @@ typedef struct {
 	TFW_HTTP_MSG_COMMON;
 	TfwStr			host;
 	TfwStr			uri_path;
+	int			uri_full;
```

Comment:
I think it's better to introduce a flag, something like `TFW_HTTP_HAS_URI` or similar, instead of extending the request structure.